pandas/core/tools/datetimes.py

```diff
@@ -965,19 +984,29 @@ def to_datetime(
 
     - However, timezone-aware inputs *with mixed time offsets* (for example
       issued from a timezone with daylight savings, such as Europe/Paris)
-      are **not successfully converted** to a :class:`DatetimeIndex`. Instead a
-      simple :class:`Index` containing :class:`datetime.datetime` objects is
-      returned:
-
-    >>> pd.to_datetime(['2020-10-25 02:00 +0200', '2020-10-25 04:00 +0100'])
+      are **not successfully converted** to a :class:`DatetimeIndex`.
+      Parsing datetimes with mixed time zones will raise a warning unless
+      `utc=True`. If you specify `utc=False` the warning below will be raised
```

Comment:
I think it's more common to talk about warnings being "shown" (and errors being "raised")

Reply:
thank you, I replaced "raise" with "show"